third_party/silabs/SiWx917_sdk.gni

```diff
@@ -299,6 +309,10 @@ template("siwx917_sdk") {
       defines += [ "ENABLE_WSTK_LEDS" ]
     }
 
+    if (sl_enable_rgb_led) {
+      defines += [ "SI917_DEVKIT" ]
+    }
```

Comment:
This condition doesn't make sense - what is the link between rgb and a 917 devkit.

Reply:
only 917 devKit BRD2605A board supports RGB LED.

Reply:
That is not exact - the MG24 thunderboard does as well. I don't think we should have this specific a define for the RBG feature.

Reply:
But as I understand RGB LED  is not been added in matter application even with mg24 thunder board. So I will be working on enabling RGB LED in matter lighting app for supported boards https://jira.silabs.com/browse/MATTER-4051